simtools/camera_efficiency.py

```diff
@@ -193,7 +183,7 @@ def analyze(self, export=True, force=False):
         """
         self._logger.info("Analyzing CameraEfficiency")
 
-        if self._file_results.exists() and not force:
+        if self._file["results"].exists() and not force:
```

Comment:
Better to use `if "results" in self._file` for dict. Also codecov complains about missing test. Maybe you can add tests? Also for the following two lines 225 and 303.

Reply:
Ok I agree. This would then probably go in a seperate PR. If you agree we can open an issue to adress this.

Reply:
Yes and no. We have all the tools like pycoverage and codecov to tell us the missing tests. Do we need to open additionally issues on that? I am really not sure.